nixos/modules/services/torrent/rtorrent.nix

```diff
@@ -180,7 +188,7 @@ in {
 
       # XMLRPC
       scgi_local = (cfg.rpcsock)
-      schedule = scgi_group,0,0,"execute.nothrow=chown,\":rtorrent\",(cfg.rpcsock)"
+      schedule = scgi_group,0,0,"execute.nothrow=chown,\":${cfg.rpcGroup}\",(cfg.rpcsock)"
```

Comment:
Probably not necessary anymore after https://github.com/NixOS/nixpkgs/pull/285299.

Reply:
If we do want to keep this separate `rpcGroup` option though, please set the `default = cfg.group` since this is a saner value.

Reply:
i see, i will implement this shortly